examples/lighting-app/efr32/src/ZclCallbacks.cpp

```diff
@@ -73,6 +76,103 @@ void emberAfPostAttributeChangeCallback(EndpointId endpoint, ClusterId clusterId
     }
 }
 
+EmberAfStatus HandleReadEthernetNetworkDiagnosticsAttribute(chip::AttributeId attributeId, uint8_t * buffer, uint16_t maxReadLength)
```

Comment:
This code is identical on all platforms. It should not be living in platform-specific files.  In fact, it should live in the relevant cluster implementation....